src/js/components/Breadcrumbs.js

```diff
@@ -0,0 +1,242 @@
+import classNames from 'classnames';
```

Comment:
I renamed the `Breadcrumb` file to `Breadcrumbs` and now it looks like a brand new file :( sorry.